Docs/user_guide/modules/retiredmodules.md

```diff
@@ -0,0 +1,11 @@
+The modules listed below have been retired and are no longer present in 3D Slicer. This page exists as a convenient marker for historical details about retired modules.
+
+Retiring modules reduces burden on slicer developers.
+
+A short list of reasons why a module may have been retired:
```

Comment:
```suggestion
A short list of reasons why a module may have been retired:

```

Reply:
@jamesobutler Adding an empty line after the `:` may help I think.

Reply:
I've pushed an update for this, but I think I had the empty line originally and it didn't work. Will see with the generated preview

Reply:
Look like the html include a list ... wondering if this is a side effect of the custom styling:

![image](https://user-images.githubusercontent.com/219043/143261205-815b8fb3-0d79-4027-aab1-ba1119f54fab.png)

It also turns out the retired page is lacking a title:

![image](https://user-images.githubusercontent.com/219043/143261295-6a704384-5c8f-460b-b62d-bd585c474e83.png)

Reply:
After providing a heading title to the new page, it appropriately renders the bullet points.